File: Algorithms/Easy/1588_SumOfAllOddLengthSubarrays/Solution.cs
public class Solution {
    public int SumOddLengthSubarrays(int[] arr) {
        int ans = 0, n = arr.Length;
        
        for (int len = 1; len <= n; len += 2) {
            int sum = 0;
            for (int i = 0; i < len; i++) {
                sum += arr[i];
            }
            ans += sum;
            for (int i = len; i < n; i++) {
                sum += (arr[i] - arr[i - len]);
                ans += sum;
            }
        }
        
        return ans;
    }
}

File: Algorithms/Easy/1588_SumOfAllOddLengthSubarrays/solution.py

from typing import List


class Solution:
    def sumOddLengthSubarrays(self, arr: List[int]) -> int:
        result = 0
        array_size = len(arr)

        for i in range(array_size):
            j = i
            while j < array_size:
                k = i
                while k <= j:
                    result = result + arr[k]
                    k += 1
                j += 2
        
        return result

